tests/scripts/thread-cert/node.py

```diff
@@ -4084,6 +4084,16 @@ def discover_mdns_service(self, instance, name, host_name, timeout=2):
                 service['addresses'] = addresses
         return service or None
 
+    def is_radvd_running(self) -> bool:
```

Comment:
Some suggestions:
- The use cases in this PR are all in this file so maybe you can make it a private method?
- The only scenario seems to throw an error when radvd is not running. If that's the case you can consider directly throwing the error from the method without returning a boolean value, i.e. repurpose it as `_assert_radvd_is_running`.